### PR TITLE
fixed typo

### PR DESCRIPTION
**Pull request by simooooone**
I fixed 2 typos in DOCS.md due to VSCodium rebranding to codium

### CODE REVIEW

**Comment by ZainW** [comment]
this should totally be merged! 

ran around for about 20 min using `vscodium` as the command

even tried to install the shell command as said in the FAQ but it didnt exist!

File: DOCS.md [hunk 1]
```diff
@@ -87,8 +87,8 @@ $ defaults write com.visualstudio.code.oss ApplePressAndHoldEnabled -bool false
 This allows you to open files or directories in VSCodium directly from your terminal:
 
 ```bash
-~/in-my-project $ vscodium . # open this directory
-~/in-my-project $ vscodium file.txt # open this file
+~/in-my-project $ codium . # open this directory
+~/in-my-project $ codium file.txt # open this file
 ```
 
 Feel free to alias this command to something easier to type in your shell profile (e.g. `alias code=vscodium`).
```
Comment (stripedpajamas) [inline DOCS.md]: This line should change as well -- `alias code=codium`

**Comment by stripedpajamas** [comment]
codium

Have just taken this screenshot -- you can update the image in the markdown to this new link: `https://user-images.githubusercontent.com/2707340/60140295-18338a00-9766-11e9-8fda-b525b6f15c13.png`

File: DOCS.md (hunk 1 above)
Comment (simooooone) [reply DOCS.md]: Edited this typo in new commit

Comment (simooooone) [reply DOCS.md]: And also the path to the image below

**Comment by stripedpajamas** [comment]
Thanks so much @simooooone

**Comment by simooooone** [comment]
It was a pleasure to work with you

**Pull request merged**